README.md

```diff
@@ -206,15 +194,15 @@ elb_loadbalancer { 'name-of-load-balancer':
     tag_name              => 'value',
   },
 }
-~~~
+```
 
-To destroy any of these resources, set `ensure => absent`.
+To destroy any of these resources, set 'ensure => absent'.
```

Comment:
chunk of code, so should be backticks.

ARGUABLY, it could be `ensure => 'absent'`, as ensure is a `setting` and 'absent' is a string, but I think that actually convolutes things.  (Partly because no one puts absent in quotes; per the style guide, they may be left bare, even though they are actually strings.)